_posts/2019-01-09-disobey-puzzle-with-powershell.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: Solving Disobey 2019 puzzle with PowerShell!
+author: Rinorragi
+excerpt: I was able to claim my hacker badge for Disobey 2019 mainly using PowerShell.
+tags:
+- disobey
+- hacking
+- infosec
+- software security
+- devsecops
+- PowerShell
+---
+
+## Disobey 
+
+[Disobey](https://disobey.fi/) is a nordic security event. It is all about gathering people from different organizations around the security topic. The atmosphere in the disobey is hacker friendly. Many are interested on the techniques to attack systems. Most of the people are white hat hackers that try to find vulnerabilities that could be then patched or limited with other measures. We have been participating Disobey in [2017](https://dev.solita.fi/2017/01/19/Disobey.html) and [2018](https://dev.solita.fi/2018/01/26/Disobey.html).
+
+## Puzzle
+
+As the event is hacker friendly it possible to get a small discount by hacking. Each year there has been a puzzle that you could solve with hacking. This years [puzzle](http://puzzle.disobey.fi/) was frustrating, fun and teached me a bunch about hacking. My coworker was able to solve it by creating some utility software with python: [https://teamrot.fi/2018/09/17/solving-the-disobey-2018-puzzle/](https://teamrot.fi/2018/09/17/solving-the-disobey-2018-puzzle/). I instead took a painful route by trying the same thing with PowerShell. It is not that PowerShell would not be good but I was kind of alone with the whole "hacking with PowerShell" mentality. Without the encouragment from [Team Rot](https://teamrot.fi) members I would not have finished it. 
```

Comment:
"friendly it possible" -> "friendly, it's possible".
"solve with hacking" -> "solve by hacking".
"This years" -> "This year's".
"encouragment" -> "encouragement".